### PR TITLE
clonehd 1.17.9 (new formula)

### PR DESCRIPTION
**Pull request by ivazquez**
cloneHD is a software for reconstructing the subclonal structure of a population from short-read sequencing data. Read depth data, B-allele count data and somatic nucleotide variant (SNV) data can be used for the inference. cloneHD can estimate the number of subclonal populations, their fractions in the sample, their individual total copy number profiles, their B-allele status and all the SNV genotypes with high resolution.

### CODE REVIEW

**Comment by ivazquez** [comment]
### Have you:

- [x] Followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-science/blob/master/.github/CONTRIBUTING.md) document?
- [x] Checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-science/pulls) for the same update/change?
- [x] Run `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting) and corrected all errors?
- [x] Built your formula locally prior to submission with `brew install <formula>`?

### New formula: have you

- [x] Written a sensible test? (The best test is to compile and run a sample program.)

**Comment by ivazquez** [comment]
Release version [upstream](https://github.com/andrej-fischer/cloneHD/releases/tag/v1.17.8) uses hard-coded g++ v4.7. Fork head avoids this and will be included in a future release.

**Comment by jonchang** [comment]
@BrewTestBot test this please

File: clonehd.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  needs :openmp
+
+  def install
+    mkdir_p "build"
+    cd "src" do
+      system "make"
+    end
+    bin.install Dir["build/*"]
+    doc.install Dir["docs/*"]
+    prefix.install Dir["data/*"]
```
Comment (jonchang) [inline clonehd.rb]: @ivazquez It looks like this line doesn't install anything? 

```
Warning: tried to install empty array to /usr/local/Cellar/clonehd/1.17.9
```

**Comment by jonchang** [comment]
Hi @ivazquez, thanks for your contribution. Once you address the comment in the diff this should be ready for merging.

File: clonehd.rb (hunk 1 above)
Comment (ivazquez) [inline clonehd.rb]: Thanks for pointing this out, @jonchang. I think my pull request is an orphan as I deleted my homebrew-science fork, so I can't push any changes. I could delete this PR and open a new one, but I think the only change that is needed as you suggest is for `prefix.install Dir["data/*"]` to point to `prefix.install Dir["test/data/*"]` instead.

**Comment by jonchang** [comment]
Closed by #4291. Thanks @ivazquez!

**Pull request closed without merging**